packages/experiments/src/components/Stack/HorizontalStack/HorizontalStack.styles.ts

```diff
@@ -34,7 +35,7 @@ export const styles = (props: IThemedProps<IHorizontalStackProps>): IHorizontalS
           display: 'block',
 
           // necessary in order to prevent collapsing margins
-          paddingTop: 1
+          padding: 1
```

Comment:
padding may be overwritten by the container or className passed in, right? I believe I did this for the stack in button.

What would happen then?

Reply:
That would be a problem if the padding is set to 0. I'll look at a workaround in a separate PR.